pkg/cli/admin/policy/modify_roles.go

```diff
@@ -218,6 +238,7 @@ func NewCmdAddClusterRoleToGroup(f kcmdutil.Factory, streams genericiooptions.IO
 		Use:   "add-cluster-role-to-group ROLE GROUP [GROUP...]",
 		Short: "Add a role to groups for all projects in the cluster",
 		Long:  addClusterRoleToGroupLongDesc,
+		Example: addClusterRoleToGroupExample,
```

Comment:
verify failure is real. Because you need to run `make update-gofmt`.

Reply:
Thanks @ardaguclu! I ran that command, let me know how it looks now